charts/trino/templates/service-worker.yaml

```diff
@@ -22,6 +22,15 @@ spec:
       protocol: TCP
       name: jmx-exporter
     {{- end }}
+    {{- range $key, $value := .Values.coordinator.additionalExposedPorts }}
```

Comment:
It looks wrong to add ports from the coordinator to the worker. Can you make sure the description of the PR explains what's the use case for this change?